### ISSUE TITLE
Add exercises to genomics and GBS sections

### ISSUE THREAD

**Issue by knausb** [grunwaldlab/Population_Genetics_in_R#26]
A comment we received more than once after the workshop as to include excercises, perhaps with an answer key, for people to work on. This would provide the examples already presented as the 'fully worked' example and provide questions that were not fully worked out for participants to test how well they learned the material.

**Comment by zkamvar** [comment]
When you implement these, if you want them to stand out from the rest of the section, you can use the CSS code @zachary-foster and I came up with for the exercise sections:

https://github.com/grunwaldlab/Reproducible-science-in-R/blob/0bb8e62689e0fb7c813051551342c046144f389b/tutorial.css#L81-L94

You can implement it in your document by defining the class for the header like so:

```
# Try it! {#exercise}
```

**Comment by sporangia** [comment]
This is a good suggestion that could (should?) be applied to the entire book.

**Comment by knausb** [comment]
Your example, @zkamvar, is greatly appreciated! We should definitely give it a try when we get around to implementing some examples.

And @sporangia 's comment is great too. Another suggestion was that SSRs were still relevant. If we wanted to expand this we could probably engineer back to back workshops that begin with SSRs and the first part of the book (that we tried to skim in this year's workshop) and followed up with an extension to genomics. Not sure how much 'workshop' everyone would be ready to tolerate though...

Thanks!

**Comment by sporangia** [comment]
I agree that SSRs are still relevant because it's neither feasible nor advised to jump directly to genomics for all situations.  I like the idea of doing a back-to-back workshop on these topics!  It would be a great mechanism to get us to finish developing the exercises.
